README.md

```
@@ -2,63 +2,90 @@
 
 # Hi, I'm **Zakaria Ait Ali**
 
-### DevOps Engineer | Cloud Architect | Platform Builder  
+### DevOps Engineer · Cloud Engineer · Platform Engineering Enthusiast
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)